Lighthouse Camera/Storyboards/LaunchScreen.storyboard

```diff
@@ -16,12 +20,21 @@
                     <view key="view" contentMode="scaleToFill" id="Ze5-6b-2t3">
                         <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
```

Comment:
The UX we are working towards has the preview and camera capture as the single point of entry for the app. So preview *is* the landing/home screen for the app (we might revise that based on user feedback)